ONSdigital/dp-web-tests
# Refactor structure

### What

Refactor the structure so that it's more modular and can scale more easily. This has meant rewriting a lot of the tests.

### How to review

1) Check that we're happy with the structure
2) Run the test:
```
./run.sh
```

### Who can review

Anyone but me.
